docs/running_grafeas.md

```diff
@@ -22,10 +22,9 @@ docker run -p 8080:8080 --name grafeas \
 
 ### Using Dockerfile
 
-To start the Grafeas server from the [Dockerfile](../Dockerfile), do:
+To start the Grafeas server from the [Dockerfile](../Dockerfile), run the following inside the repository folder:
```

Comment:
One way to make this clear is just include the cd command.

Reply:
Good point, done.